doc/releases.rst

```diff
@@ -0,0 +1,121 @@
+Release history
+###############
+
+Release notes
+*************
+
+QISKit SDK 0.5.0
+================
+
+This release brings a number of improvements to QISKit, both for end users and
```

Comment:
I would say: `...both for the user experience and under...` since the "hidden changes" are also for the end user sake.

Reply:
:+1: